docs/structure-samples/csharp/csharp.md

```diff
@@ -0,0 +1,54 @@
+## C# and .NET project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a C#/.NET standard application.
+
+```bash
+~root
+│
+├── .devcontainer.json          # Moved to top
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml
+│       ├── workflow2.yml
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep
+│   ├── main.parameters.bicep
+│   ├── abbreviations.json
+│   └── ...                         
+│
+├── src/
+│   ├── ProjectName.Core/
```

Comment:
If we want to be detailed in the project name structure in this document, Here's what I'm thinking:

Project.API - uses Minimal API to define backend APIs for the sample
Project.Web - (Optional) Blazor Server-Side UI for testing/working with the API
Project.SharedModel - (Optional) Model objects that are shared between the API and optional .NET front-end.

@cassiebreviu, @luisquintanilla   - thoughts?

Reply:
Any resolution?